ocs_ci/ocs/cluster.py

```diff
@@ -500,6 +500,85 @@ def enable_health_monitor(self, sleep=5):
     def disable_health_monitor(self):
         self.health_monitor_enabled = False
 
+    def get_ceph_cluster_iops(self):
+        """
+        The function gets the IOPS from the ocs cluster
+
+        Returns:
+            Total IOPS in the cluster
+        """
+
+        ceph_status = self.get_ceph_status()
+        logger.info(ceph_status)
+        for item in ceph_status.split("\n"):
+            if 'client' in item:
+                iops = re.findall(r'\d+\.+\d+|\d\d*', item.strip())
+                iops = iops[2::1]
+                if len(iops) == 2:
+                    iops_in_cluster = float(iops[0]) + float(iops[1])
+                else:
+                    iops_in_cluster = float(iops[0])
```

Comment:
Tried testing this code ?, if any help required let me know.

Reply:
@ramkiperiy Tested the code and attached the logs below. Please have a look at it and let me know if something had to be improved.